README.md

```diff
@@ -0,0 +1,58 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Personal Potfolio
+
+> This project is about Dameh Seko as a web developer. I am very passoinate about software. Get in touch with me through any of my handles listed below and i will be of help. See the technologies i can use below. 
+
```

Comment:
This place is supposed to bear the description of the project. Kindly make sure you put that into consideration.